pkg/machine/hyperv/machine.go

```diff
@@ -555,15 +555,15 @@ func getVMConfigPath(configDir, vmName string) string {
 	return filepath.Join(configDir, fmt.Sprintf("%s.json", vmName))
 }
 
-func loadMacMachineFromJSON(fqConfigPath string, macMachine *HyperVMachine) error {
+func loadHyperVMachineFromJSON(fqConfigPath string, hypervMachine *HyperVMachine) error {
```

Comment:
I think this should be a methon of HyperVMachine:
```golang
func (h *HyperVMachine) loadFromJSON(configPath string) error
````

As mentioned in other comments, I really dislike IN/OUT parameters.  They are less idiomatic and since we already control the HyperVMachine type, we can make use of golang's type system.

Reply:
Fixed.